v22.2/backup.md

```diff
@@ -337,6 +337,8 @@ BACKUP DATABASE movr INTO 's3://{BUCKET NAME}?AWS_ACCESS_KEY_ID={KEY ID}&AWS_SEC
 
 <section class="filter-content" markdown="1" data-scope="azure">
 
+{% include {{ page.version.version }}/backups/azure-url-encode.md %}
```

Comment:
Yes agreed, the filters aren't needed. Should we be removing all of this info from each section then?

Reply:
You can either:

-  Let your changes stand in the filter sections and merge, since you've already added it. Once my PR merges later this week, or next, it will just remove any of these notes. (I can reshuffle things around as necessary.)
OR
- Remove any changes you have made in this PR to the filter sections and just merge changes to the Use Cloud for Bulk Operations page.

Reply:
Going with option 1, good to have you remove these notes when you remove the filters!